lessons/topics/01 - Variables/README.md

```diff
@@ -0,0 +1,18 @@
+# Variables, Scope and Hoisting
+
+In this chapter we will talk about how variables behave in different contexts, how the scope affect the reach of said chunks of data and how we can manage to move the data between blocks and structures
+
+## Previous Knowledge
```

Comment:
same here. I don't understand why previous knowledge that we will explain should be required before explaining the concepts.